BeagleBone_black/src/state_machine/main.cpp

```diff
@@ -62,12 +64,12 @@ void Main::checkNavigation()
   *  @TODO Check if margin (20m) is appropriate
   */
 
-if((nav_data_.distance + nav_data_.emergency_braking_distance) + 20 >= comms_data_.run_length)
+if ((nav_data_.distance + nav_data_.emergency_braking_distance) + 20 >= comms_data_.run_length)
```

Comment:
also add proper indentation (that applies for several other places as well)

Reply:
you need to check the values you are using are valid, eg. by checking `nav_data.module_status != kStart`